cli/cycode.py

```diff
@@ -60,8 +61,13 @@
               the default is text
               """,
               type=click.Choice(['text', 'json']))
+@click.option('--min-severity',
+              default=None,
+              help='Specify SCA packages vulnerabilities violations minimum severity',
```

Comment:
please consult with Maria on a text. need to contain:
1. list of the severities
2. we only show the violation of the specified severity level or higher
3. only supported for SCA scan (actually can be supported to all scans?)

Reply:
1. we already have
Options:
  -t, --scan-type [secret|iac|sca]
                                  Specify the scan you wish to execute (secret/iac/sca), 
                                  the default is secret
  --secret TEXT                   Specify a Cycode client secret for this
                                  specific scan execution
  --client-id TEXT                Specify a Cycode client ID for this specific
                                  scan execution
  --show-secret                   Show secrets in plain text
  --soft-fail                     Run scan without failing, always return a
                                  non-error status code
  --output [text|json]            Specify the results output (text/json), 
                                  the default is text
  --severity-threshold [INFO|LOW|MEDIUM|HIGH|CRITICAL]
                                  Specify SCA packages vulnerabilities
                                  violations minimum severity
  --help                          Show this message and exit.

Reply:
Agree, especially with 3#. 
It should be supported for all the scans types.

Reply:
@MaorDavidzon it's a little bit issue because we dont have the detection severity of the reponse.
in sca we hav the severity onside the detection details.

Reply:
@avishaiamiel but in other scans we return the severity?

Reply:
as been discussed, the severity is not part of detection message. hence, we dont have it in our hand.